r_scripts/analysis/oregon_part_a.Rmd

```diff
@@ -0,0 +1,1083 @@
+                  "Heart Failure", "Cerebrovascular Disease")), 1, 0)) %>% 
+    # filter out rare events that have unstable ORs
+  filter(flag == 0) %>% 
+  select(outcome, strata, odds_ratio:upper95) 
+
+# total strata
+total_results <- read_csv(paste0("./all_strata_result_table.csv")) %>% 
+  mutate(strata = "All Strata") %>% 
+  select(outcome, strata, odds_ratio:upper95) 
+
+# sex strata
+sex_results <- read_csv(paste0("./sex_result_table.csv")) %>% 
+  mutate(strata = case_when(sex == 0 ~ "Male",
+                            sex == 1 ~ "Female")) %>% 
+  select(outcome, strata, odds_ratio:upper95) 
+
+
+# rowbind
+results <- bind_rows(total_results, sex_results, age_results) %>% 
+  # filter to certain outcomes
+  filter(outcome %in% c("All Respiratory", "Asthma", "COPD", 
+                        "Cardiovascular Disease", "Ischemic Heart Disease",
+                        "Heart Failure", "Cerebrovascular Disease")) %>% 
+  # set order of strata levels
+  mutate(outcome = factor(outcome, levels = c("All Respiratory", "Asthma", "COPD", 
+                        "Cardiovascular Disease", "Ischemic Heart Disease",
+                        "Heart Failure", "Cerebrovascular Disease")),
+         strata = factor(strata, levels = c("All Strata", "Female", "Male",
+                                            "Age < 15", "Age 15-65", "Age > 65")),
+         # group cvd and respiratory outcomes
+         Group = as.factor(if_else(outcome %in% c("All Respiratory", "Asthma", "COPD"), 
+                         "Respiratory", "Cardiovascular Disease")))
+
+result_plot <- ggplot(results, aes(x = outcome, y = odds_ratio, color = Group)) +
+  geom_point() + 
+  geom_errorbar(aes(ymin=lower95, ymax=upper95), width = 0.2) +
+  geom_hline(yintercept = 1, linetype=2) +
+  scale_color_manual(values = c("red", "blue")) +
+  facet_wrap(~strata) +
+  ylab(expression(paste("Odds Ratio for 10�g/m"^3, " Increase in PM"[2.5]))) +
+  xlab("Outcome") +
+  # plot theme
+  theme(panel.background = element_rect(fill = 'white', colour = 'black'),
+        panel.grid.major = element_blank(),
+        panel.grid.minor = element_blank(),
+        # strip element
+        strip.background = element_rect(colour=NA, fill=NA),
+        panel.border = element_rect(fill = NA, color = "black"),
+        # facet text size
+        strip.text = element_text(size = 10),
+        # axis element (tilt)
+        axis.text.x = element_text(angle = 45, hjust =1),
+        #axis.title.x = element_blank(),
+        #axis.ticks.x = element_blank(),
+        axis.title.y = element_text(angle = 90))
+
+result_plot
+
+# print plot
+ggsave("../plot_new/cardiopulmonary_er_result.tiff", plot = result_plot, units="in", width=12, height=8, dpi=400)
+
+# ggsave("../plot_new/cardiopulmonary_er_result_table.pdf", plot = result_plot, 
+#         width = 12, height = 8, units = "in", dpi = 400)
+
+tiff('../plot_new/cardiopulmonary_er_result.tiff', units="in", width=12, height=8, res=300)

```

Comment:
12 is kind of a big width. Can you make it width 8 and height 6, res 300? This will be easier to upload to a journal. Also choose either ggsave or tiff.